lib/Doctrine/Common/DataFixtures/Loader.php

```diff
@@ -322,6 +322,10 @@ private function getUnsequencedClasses($sequences, $classes = null)
         }
 
         foreach ($classes as $class) {
+            if(!isset($sequences[$class])) {
+                dump('UNDEFINED INDEX');
+                die;
```

Comment:
this should be removed, as it prevents merging the new test in the existing testsuite